backend/router.ts

```diff
@@ -0,0 +1,82 @@
+import { Router, Request, Response} from "express";

```

Comment:
Sometimes you have spaces in square brackets, sometimes not. It would be better to have them (or not) in entire project to be consistent. ESLint would help you with that.